### PR TITLE
Add link to virtualenv < 20.0.0 changelog.

### PR DESCRIPTION
**Pull request by jezdez**
## Thanks for contributing a pull request, see checklist all is good!

- [x] updated/extended the documentation

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -73,4 +73,4 @@ v20.0.0b1 (2020-01-28)
 
    The current virtualenv is the second iteration of implementation. From version ``0.8`` all the way to ``16.7.9``
    we numbered the first iteration. Version ``20.0.0b1`` is a complete rewrite of the package, and as such this release
-   history starts from there.
+   history starts from there. The old changelog is still available in the `virtualenv < 20.0.0 documentation <https://virtualenv.pypa.io/en/16.7.9/changes.html>`_.
```
Comment (gaborbernat) [inline docs/changelog.rst]: Let's use the legacy name, see https://virtualenv.pypa.io/en/legacy/

Comment (pradyunsg) [inline docs/changelog.rst]:
```suggestion
   history starts from there. The old changelog is still available in virtualenv's `legacy documentation <https://virtualenv.pypa.io/en/legacy/changes.html>`_.
```

Comment (gaborbernat) [reply docs/changelog.rst]: seems I'm not allowed to push to @jezdez repository, so can't accept suggested changes

Comment (jezdez) [reply docs/changelog.rst]: Good idea!

Comment (jezdez) [reply docs/changelog.rst]: Woops, sorry, only seeing this now :)

**Pull request merged**